src/pages/resources/test_your_skills.mdx

```diff
@@ -4,10 +4,16 @@ description: The best way to cement your skill is to test your knowledge in prac
 layout: ../../layouts/MainLayout.astro
 ---
 
+## Platforms for testing your skills
+
+import Platforms from '../resources/_exercises/frontend_platforms.md'
+
+<Platforms />
+
 ## Mocked interview
 
-- [Conner Ardman - React Coding Interview Ft. Clément Mihailescu (build the wordle clone)](https://www.youtube.com/watch?v=5xf4_Kx7azg)
-  - [Used API (if someone wants to recreate the task)](https://api.frontendexpert.io/api/fe/wordle-words)
+[Conner Ardman - React Coding Interview Ft. Clément Mihailescu (build the wordle clone)](https://www.youtube.com/watch?v=5xf4_Kx7azg)
```

Comment:
Make this a list, otherwise it will look like this:
![obraz](https://user-images.githubusercontent.com/25301582/232431825-584dfc31-bdf0-45c7-a37d-37e9b37534ae.png)

Reply:
Just two newlines will fix that, I removed the list, because it's only one item

Reply:
Here
![image](https://user-images.githubusercontent.com/55458485/232432687-9b125609-aa1e-40d3-9b5e-796380e9a4f4.png)